### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by Padraic-O-Mhuiris**

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Quick Install
+=================================
+
+Blah
```
Comment (dpaiton) [inline docs/source/installation.rst:4]: Is this rendered in the docs? Would rather not have the blah blah blah show up anywhere :-P

Comment (dpaiton) [reply docs/source/installation.rst:4]: You can just point them to the readme for now

Comment (Padraic-O-Mhuiris) [reply docs/source/installation.rst:4]: It was just temp - I'll clean it up

**Review by dpaiton — APPROVED** [review]
Thanks, this is a huge improvement!

**Pull request merged**